bazel.el

```diff
@@ -1915,12 +1915,18 @@ Return nil if no .bazelignore file exists."
   "Run Bazel test TARGET with coverage instrumentation enabled."
   (interactive (list (bazel--read-target-pattern "coverage" :only-tests)))
   (cl-check-type target string)
-  (bazel--run-bazel-command "coverage" target)
+  (bazel "coverage" target)
   nil)
 
-(defun bazel--run-bazel-command (command target-pattern)
+(defun bazel (command target-pattern)
```

Comment:
With everything being bazel-related in the project, it's not obvious to know what the functino `bazel` does.
Maybe `bazel-custom-command`? Or `bazel-command`?

At the same time, the command doesn't seem to be very generic, with the commands `query`, `clean`, `info`, etc. not allowed.